native/src/components/TtsContainer.tsx

```diff
@@ -1,31 +1,41 @@
-import React, { createContext, ReactElement, useCallback, useEffect, useMemo, useState } from 'react'
+import React, { createContext, ReactElement, useCallback, useContext, useMemo, useState } from 'react'
 import { useTranslation } from 'react-i18next'
-import { AppState, Platform } from 'react-native'
-import Tts from 'react-native-tts'
+import Tts, { Options } from 'react-native-tts'
 
 import { truncate } from 'shared/utils/getExcerpt'
 
 import buildConfig from '../constants/buildConfig'
 import { AppContext } from '../contexts/AppContextProvider'
+import useAppStateListener from '../hooks/useAppStateListener'
+import useSnackbar from '../hooks/useSnackbar'
 import { reportError } from '../utils/sentry'
 import TtsPlayer from './TtsPlayer'
 
 const MAX_TITLE_DISPLAY_CHARS = 20
+const TTS_UNSUPPORTED_LANGUAGES = ['fa', 'ka', 'kmr']
+const TTS_OPTIONS: Options = {
+  androidParams: {
+    KEY_PARAM_PAN: 0,
+    KEY_PARAM_VOLUME: 0.6,
+    KEY_PARAM_STREAM: 'STREAM_MUSIC',
+  },
+  iosVoiceId: '',
+  // This must not be 1 on iOS
```

Comment:
@f1sh1918 do you have an issue link for this?

Reply:
No